src/core/public/i18n/i18n_eui_mapping.tsx

```diff
@@ -500,10 +519,13 @@ export const getEuiContextMapping = () => {
     'euiFilePicker.filesSelected': i18n.translate('core.euiFilePicker.filesSelected', {
       defaultMessage: 'files selected',
     }),
+    'euiFilePicker.removeSelected': i18n.translate('core.euiFilePicker.removeSelected', {
+      defaultMessage: 'Remove',
+    }),
     'euiFilterButton.filterBadge': ({ count, hasActiveFilters }: EuiValues) =>
       i18n.translate('core.euiFilterButton.filterBadge', {
-        defaultMessage: '${count} ${filterCountLabel} filters',
-        values: { count, filterCountLabel: hasActiveFilters ? 'active' : 'available' },
+        defaultMessage: '{count} {hasActiveFilters} filters',
+        values: { count, hasActiveFilters: hasActiveFilters ? 'active' : 'available' },
```

Comment:
I don't think we should be doing that `'active' | 'available'` text replacement here. They'll show as-is for any other locales.